src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -2645,6 +2652,12 @@ private void EmitSizeOfExpression(BoundSizeOfOperator boundSizeOfOperator)
             EmitSymbolToken(type, boundSizeOfOperator.SourceType.Syntax);
         }
 
+        private void EmitMethodTokenExpression(BoundMethodToken node)
+        {
+            _builder.EmitOpCode(ILOpCode.LoadMethodTokenAsInteger_i4);
```

Comment:
LoadMethodTokenAsInteger_i4 is a bogus/pseudo opcode that is rewritten during metadata writing to become a Ldc_i4 instruction with the real metadata token of the method as its value.